packages/lib/server/repository/booking.ts

```diff
@@ -0,0 +1,11 @@
+import { prisma } from "@calcom/prisma";
+
+export class BookingRepository {
+  static async getBookingAttendees(bookingId: number) {
+    return await prisma.attendee.findMany({
+      where: {
+        bookingId,
+      },
+    });
+  }
+}
```

Comment:
do we really need this class? you have not defined any select field here

Reply:
While it would be nice to have a full flushed out `BookingRepository` we can add methods to this one by one and slowly transition over. When getting resources we don't need to add select fields here as it's a small record.